WatchParty/backend/Procfile

```diff
@@ -0,0 +1 @@
+web: node bin/www
```

Comment:
Hmmm, heroku runs script.start by default

Reply:
The `Procfile` is Heroku's convention of specifying an app's entry point. Here are the Heroku docs for `Procfile` https://devcenter.heroku.com/articles/getting-started-with-nodejs#define-a-procfile. I set up `npm start` to be more for development, but I can see how that's little unclear. Should I separate the `npm` scripts for development and production?

Reply:
Right, I need to look over this more I didn't notice it gives priority in the js file to the env port, that tripped me up a bit